docs/setup_help/m1_mac_setup.md

```diff
@@ -0,0 +1,47 @@
+#Notes from Wes Goar on setup of vrs-python on a m1 mac
+- @author Wes Goar
+- date: 20220328
+- macOS version: 12.3
+- python version: 3.9.11
+
+##Notes
+- I installed seqrepo before trying this setup.
+- I had already installed xcode and homebrew on my machine.
+
+###Steps
+1. brew install openssl
+2. add the following statement in your .zshrc: export PATH="/opt/homebrew/opt/openssl@1.1/bin:$PATH"
```

Comment:
This is specific to z shell. Should make a note for that if users are using a different shell.

Reply:
Noted.